quinn-udp/src/windows.rs

```diff
@@ -140,8 +155,240 @@ impl UdpSocketState {
     }
 }
 
+fn send(
```

Comment:
What's the value in this indirection?

Reply:
it come from the time where we had both implementation (new one using `wsasendmsg` and old one using send_to), i can flatten the code.